apps/web/src/services/asset-inventory/collector/modules/CollectorItemInfo.vue

```diff
@@ -171,53 +176,54 @@ const props = withDefaults(defineProps<Props>(), {
 const collectorPageStore = useCollectorPageStore();
 const collectorPageState = collectorPageStore.$state;
 
-const storeState = reactive({
+const state = reactive({
     schedule: computed(() => collectorPageState.schedules.find((schedule) => schedule.collector_info.collector_id === props.item.collectorId)),
-});
+    collectorState: computed(() => state.schedule?.collector_info.state),
+    toggleStatus: computed(() => (state.collectorState === COLLECTOR_STATE.ENABLED ? 'ON' : 'OFF')),
+    nextSchedule: computed(() => {
+        if (state.schedule) {
+            const numbersArray = state.schedule.schedule.hours;
```

Comment:
question: Does `schedule.hours` always exist? If it does not, please guard it like below:

```suggestion
const numbersArray = state.schedule.schedule?.hours ?? [];
```